<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEpicsPvGUI</class>
 <widget class="QWidget" name="QEpicsPvGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QMLineEdit" name="pv">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="set">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="lineW">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QMLineEdit" name="lineBox">
         <property name="text">
          <string>Disconnected</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="doubleW">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QMDoubleSpinBox" name="doubleBox">
         <property name="decimals">
          <number>9</number>
         </property>
         <property name="minimum">
          <double>-2147483647.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2147483647.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="enumW">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QMComboBox" name="enumBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="intW">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QMSpinBox" name="intBox">
         <property name="minimum">
          <number>-2147483647</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="get">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Disconnected</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qtpv_guielements.h</header>
   <slots>
    <signal>valueEdited(double)</signal>
    <signal>escaped()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QMSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qtpv_guielements.h</header>
  </customwidget>
  <customwidget>
   <class>QMComboBox</class>
   <extends>QComboBox</extends>
   <header>qtpv_guielements.h</header>
  </customwidget>
  <customwidget>
   <class>QMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qtpv_guielements.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
